dapps/digital-twin-detail/src/components/container/List.ts

```diff
@@ -34,6 +34,15 @@ interface File {
   name: string;
   size: number;
 }
+
+interface ColumnInterface {
```

Comment:
This could be moved to the existing EvanTableInterfaces.ts in the evancore to use it everywhere :)